datamodel/app/functions/meta_functions.sql

```diff
@@ -0,0 +1,20 @@
+CREATE OR REPLACE FUNCTION tww_app.check_all_nulls(rec record, prefix varchar(2),ignored_postfix text[] default Array['obj_id','identifier'])
```

Comment:
Can we have a comment or some more explanation of what does this function exactly?

Reply:
It allows checking a record (usually NEW) on whether or not its attributes with a certain prefix are all NULL. The third argument allows whitelisting auto-populated values